packages/components/src/navigation/stories/index.js

```diff
@@ -28,17 +28,28 @@ const data = [
 	{
 		title: 'Category',
 		id: 'item-3',
+		badge: '2',
 	},
 	{
 		title: 'Child 1',
 		id: 'child-1',
 		parent: 'item-3',
+		badge: '1',
 	},
 	{
 		title: 'Child 2',
 		id: 'child-2',
 		parent: 'item-3',
 	},
+	{
+		title: 'Custom link',
+		id: 'item-4',
+		linkTag: 'a',
```

Comment:
Can we make an assumption here that if an `href` is found, use a anchor tag?

Reply:
Now that I think about it, `Button` already has this logic built in.
https://github.com/WordPress/gutenberg/blob/375eaa0d870282e12cd461a53463d3e10e776c41/packages/components/src/button/index.js#L67

Reply:
Ya, button already includes this logic; the use of `a` here is purely for demonstration, though we could easily swap this with another tag if you think it would be more appropriate.

Using `linkTag` allows us to pass a prop like the `Link` component or some other custom component to integrate with various routers.